pkg/operator/configobservation/interfaces.go

```diff
@@ -26,6 +30,10 @@ func (l Listers) FeatureGateLister() configlistersv1.FeatureGateLister {
 	return l.FeatureGateLister_
```

Comment:
what is the `_` ?

Reply:
this line is not part of my diff but the idea is to avoid collision with a function that has same name. I kinda cargo-culted @deads2k example from before.